public/js/presenter.js

```diff
@@ -800,7 +800,7 @@ function chooseLayout(layout)
 
     case 'beside':
       $('#preview').removeClass('beside');
-      $('#preview #nextSlide').removeAttr("style");
+      $('#preview #nextSlide .container').removeAttr("style");
```

Comment:
Not for now, but at some point we should store #preview as a local variable instead of making repeated calls to the DOM.  Would be faster